ask-sdk-core/lib/delegation/DelegationData.ts

```diff
@@ -0,0 +1,17 @@
+import { Intent, Slot } from "ask-sdk-model";
+
+export interface DelegationData {
+    delegateTarget: DelegateTarget;
+    intentData? : Intent;
+    acirData? : any;
+}
+
+export interface DelegateTarget {
+    type: string;
+    name: string;
```

Comment:
Since we have some pre-defined delegate targets so we can have this name as an ENUM.

Reply:
Created a type target